db/seeds.rb

```diff
@@ -5,4 +5,13 @@
 User.find_or_create_by!(email: EMAIL) do |user|
   user.password = PASSWORD
   puts 'ユーザーの初期データインポートに成功しました。'
-end
\ No newline at end of file
+end
+
+EMAIL = 'admin@example.com'
+PASSWORD = 'password'
```

Comment:
定数名を変更して3行目に移動させましょう。

```suggestion
ADMIN_EMAIL = 'admin@example.com'
ADMIN_PASSWORD = 'password'
```